assets/common/components/mlflow_model_local_validation/spec.yaml

```diff
@@ -35,7 +36,24 @@ inputs:
     optional: true
     description: |
       Provide mapping of dataset column names that should be renamed before inferencing.
-      eg: col1:ren1; col2:ren2; col3:ren3 
+      eg: col1:ren1; col2:ren2; col3:ren3
+
+  task_name:
+    description: A Hugging face task on which model was trained on
+    enum:
+      - text-classification
```

Comment:
what about vision model releated tasks
image-object-detection and image-instance-segmentation

Reply:
got it added in ES.